GlobalStaticVersion.props

```diff
@@ -8,12 +8,12 @@
     <SemanticVersionMajor>2</SemanticVersionMajor>
     <SemanticVersionMinor>2</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
-    <PreReleaseMilestone>beta6</PreReleaseMilestone>
+    <PreReleaseMilestone></PreReleaseMilestone>
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-06-02</SemanticVersionDate>
+    <SemanticVersionDate>2016-12-05</SemanticVersionDate>
```

Comment:
this will crew up file versions. Please do not do it before the major or minor version change. Like 2.3 should have this date increased